src/index.js

```diff
@@ -30,6 +31,7 @@ const HANDLERS = {
   'lhs-desktop': lhs,
   404: notfound,
   'broken-backlinks': backlinks,
+  'organic-keywords': organicKeywords,
```

Comment:
For all of the keys here, we could probably reuse the types exported from the audit model